### DISCUSSION TITLE
nx daemon is going to be disabled until you run "nx reset"

### DISCUSSION THREAD

**Discussion by HunagYouJun** [gperdomor/oss#454] [Q&A]
Dockerfile
```
#run web build & run
FROM node:16-alpine3.14
COPY . .

#install node_modules
RUN npm i
RUN npm install -g nx

#build project
RUN nx build
RUN nx build web-server

#run server
CMD node dist/apps/web-server/main.js
EXPOSE 3333
```

> i run nx docker, but cmd stay on nx daemon

#10 [5/6] RUN nx build
#10 4.149 
#10 4.155  >  NX   Nx Daemon was not able to compute the project graph.
#10 4.155 
#10 4.155    Log file with the error: /node_modules/.cache/nx/d/daemon-error.log
#10 4.155    Please file an issue at https://github.com/nrwl/nx
#10 4.155    Nx Daemon is going to be disabled until you run "nx reset".
#10 4.155

**Comment by vinod-amt** [comment]
i have a same issue.
